cli/common/lua.go

```diff
@@ -67,7 +67,7 @@ func LuaGetRocksVersions(appDirPath string) (map[string]string, error) {
 				depInfoLTable.ForEach(func(depVersionL lua.LValue, _ lua.LValue) {
 					depVersion := depVersionL.String()
 					if _, found := rocksVersionsMap[depName]; found {
-						log.Warnf(
```

Comment:
It should be a warning on the pack.

Reply:
Probably, on `cartridge version` we should show warnings too, but make it correctly. BTW, can it be one warning with a list of duplicated rocks?